hubspot/decorators.py

```diff
@@ -0,0 +1,34 @@
+"""
+Decorators for hubspot module
+"""
+import functools
+from time import sleep
+
+from requests import HTTPError
+
+
+def try_again(func):
+    """
+    Wrapper for sending requests to hubspot. Attempts to send requests multiple times.
+    Hubspot does not seem to return 429 errors so any error is considered a too many requests error for the purposes
+    of try again.
+
+    The wrapped function should return a requests response object.
+    """
+    @functools.wraps(func)
+    def wrapper_try_again(*args, **kwargs):
+        max_attempts = 3
```

Comment:
Maybe (in a followup PR, this is fine for now) this could be configurable via a .env/request setting, so it could be tweaked without a code change.  Something like
```python
max_attempts = settings.HUBSPOT_MAX_RETRIES
```